src/signature.c

```diff
@@ -1133,7 +1133,9 @@ static void debug_cms_ci(CMS_ContentInfo *cms)
 	gchar *out_str = NULL;
 	long size;
 
-	if ((domains == NULL) || (strstr(domains, G_LOG_DOMAIN "-signature") == NULL)) {
+	if (domains == NULL)
+		return;
+	if ((strstr(domains, "all") == NULL) && (strstr(domains, G_LOG_DOMAIN "-signature") == NULL)) {
```

Comment:
In the way glib and RAUC interpret the domains, `"all"` should only be used as a single element. Thus using `g_strcmp0()` or `g_str_equal()` should be preferred here. It would also ensure that no domain name that accidentally contains `"all"` as substring would match.

Reply:
You made me take a closer look at the GLib documentation. And there I found this note:

> It is recommended that custom log writer functions re-use the G_MESSAGES_DEBUG environment variable, rather than inventing a custom one, so that developers can re-use the same debugging techniques and tools across projects. Since GLib 2.68, this can be implemented by dropping messages for which g_log_writer_default_would_drop() returns TRUE.

I will revise my commit to use `g_log_writer_default_would_drop()`

Reply:
We still support GLib 2.56, so we can use `g_log_writer_default_would_drop()` yet.

Reply:
Oh I see